packages/core/src/sync-store/store.test.ts

```diff
@@ -57,7 +57,7 @@ test("insertLogFilterInterval inserts block, transactions, and logs", async (con
 
   await syncStore.insertLogFilterInterval({
     chainId: 1,
-    logFilter: { address: erc20.address },
+    logFilter: { address: erc20.address, topics: [null, null, null, null] },
```

Comment:
Do we need the `null`s here? Could this just be an empty array?

Reply:
yes